modules/accounts/contacts.nix

```diff
@@ -55,23 +56,21 @@ let
         description = "User name for authentication.";
       };
 
-      userNameCommand = mkOption {
-        type = types.nullOr (types.listOf types.str);
-        default = null;
-        example = [ "~/get-username.sh" ];
-        description = ''
-          A command that prints the user name to standard
-          output.
-        '';
-      };
+      # userNameCommand = mkOption {
+      #   type = types.nullOr (types.listOf types.str);
+      #   default = null;
+      #   example = [ "~/get-username.sh" ];
+      #   description = ''
+      #     A command that prints the user name to standard output.
+      #   '';
+      # };
```

Comment:
The same as for [modules/accounts/calendar.nix line 59ff](https://github.com/rycee/home-manager/blob/8450842e4957915243decef882f3cece2b54463d/modules/accounts/calendar.nix#L59) (see above) applies here.